docs/content/concepts/layer/implementation/ssp/_index.md

```diff
@@ -66,6 +66,46 @@ The figure below expresses represents the portion of the OSCAL stack as it relat
 
 {{<partialCached "note-to-developers-uuid.html" >}}
 
+### Identifier References
+
+An OSCAL SSP may contain references to information that is defined locally (e.g., in the SSP model) or externally (e.g., in a referenced profile, catalog, or component definition model).  The following lists the mechanisms through which objects in an OSCAL SSP may "link" to other OSCAL content:
+
+- **Local**: Scenario where an identifier references information in the same OSCAL instance.  This scenario supports  “compile time” enforcement of constraints since the validity of the reference can be checked as soon as the SSP is authored.
```

Comment:
I suggest classifying the references as **local** and **external** with different use cases for the **external** type of references: 1) in another OSCAL instance like 1.1) an imported profile,  1.2) in a leveraged authorization, 1.3) in a cdef sourced by a component of a system or 2) in a document referenced by a general link.

Reply:
We should use *local* and *cross-instance* to be consistent with the [identifier scoping](https://pages.nist.gov/OSCAL/concepts/identifier-use/#scope). *external* should be limited to references to external non-OSCAL resources.

Reply:
Created subsection headers for **instance**, **cross instance**, and **external** with narrative scenario descriptions.